odl/solvers/functional/functional.py

```diff
@@ -822,6 +823,83 @@ def __str__(self):
                                           self.translation)
 
 
+class InfimalConvolution(Functional):
+
+    """The functional representing ``h(x) = inf_y f(x-y) + g(y)``."""
```

Comment:
My "the" was actually copy pasting from "FunctionalQuadraticPerturb". How would one correct these typos? Probably not with a pull request for every typo?